creator-node/src/routes/files.js

```diff
@@ -460,10 +498,21 @@ const getCID = async (req, res) => {
   const blockStartMs = Date.now()
   try {
     startMs = Date.now()
-    const found = await findCIDInNetwork(storagePath, CID, req.logger, trackId)
-    if (!found) {
-      throw new Error('Not found in network')
+    const error = await saveFileForMultihashToFS(
+      libs,
+      req.logger,
+      CID,
+      null /* dirCID */,
+      [] /* targetGateways */ // no replica set so it will scan the whole network
+    )
+    if (error) {
+      throw new Error(`Not found in network: ${error.message}`)
     }
+
+    // need to recompute the storagePath to stream the file because
+    // saveFileForMultihashFromFS recomputes storagePath
+    storagePath = DiskManager.computeFilePath(CID)
```

Comment:
fine to do here but feels redundant since saveFileForMutihash already computed this and could just return the destination path? like return format is { error?, storagePath? } or smth